prime-router/src/main/kotlin/fhirengine/azure/FHIRFlowFunctions.kt

```diff
@@ -54,8 +61,20 @@ class FHIRFlowFunctions : Logging {
             responseBuilder.body(body)
         } catch (e: IllegalArgumentException) {
             responseBuilder.body(e.message)
+            actionHistory.trackLogs(
```

Comment:
Should the exception be logged?  otherwise the stack trace is lost.

Reply:
I think it would be good to track the exception as well. Please log it.

Reply:
Like the whole e object? I can do that, just want to be sure before I make the change.

Reply:
Looks like stackTraceToString is the method that converts the whole thing to a string. Incoming commit will do that.